src/Email/SendNotificationDefinitionConfirmation.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Email;
+
+use App\Entity\NotificationDefinition;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+class SendNotificationDefinitionConfirmation
```

Comment:
Obie te 2 klasy w katalogu Email proponuje przenieść do Symfony Messenger:

https://symfony.com/doc/current/messenger.html

Póki co możesz mieć transport "sync", a w przyszłości można wpiąć kolejkę (np rabbitmq)

Reply:
Jak użyjesz symfony messenger to w message nie przekazuj obiektu NotificatioDefinition tylko samo id i w message handler pobieraj z repository tą definicję. Tutaj masz wyjaśnienie: https://symfony.com/doc/current/messenger.html#doctrine-entities-in-messages